### PR TITLE
Feature/jjdd8fpo 19 a

### PR DESCRIPTION
**Pull request by adamprzybylowski**
"PolishLanguage.propertise -DONE, EngLan.prop -in progress

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/com/infoshareacademy/FavoriteBooks.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.infoshareacademy;
+
+public class FavoriteBooks {
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/FavoriteBooks.java]: po co to ?

File: src/main/resources/EnglishLanguage.properties [hunk 2]
```diff
@@ -0,0 +1,35 @@
+main_menu = "Main menu"
```
Comment (rafalmisiak) [inline src/main/resources/EnglishLanguage.properties]: do skasowania spacja przed = i po =
do skasowania "

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/resources/EnglishLanguage.properties [hunk 3]
```diff
@@ -0,0 +1,70 @@
+main_menu =Main menu
```
Comment (rafalmisiak) [inline src/main/resources/EnglishLanguage.properties]: masz złe nazwy plików, skonsultuj się z @michal-brakon

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by adamprzybylowski** [comment]
messages.propertise -DONE (Polish)  (English inprogress)

**Pull request merged**